mkversion.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+cd "$(dirname "$0")"
+isdeb=true
+v="$( head -n1 debian/changelog  | sed -r -e 's/^[^ ]+ \((.*)\)( +[^ ]+)+; urgency=.*/\1/' )"
+if [ -z "$DH_GOPKG" ]; then
+    isdeb=false
+    # not building the package; add something to the version
+    if git rev-parse --is-inside-work-tree >&- 2>&-; then
+        # in git; good.
+        v="$( printf "%s+git%s.%04d" "$v" "$( date --utc +%Y%m%d )" "$( git log --since=yesterday --pretty=short | wc -l)" )"
```

Comment:
I guess the aim of `git log --since=yesterday --pretty=short|wc -l` is to increase for each commit since yesterday? short has 4 lines per commit, if my assumption is right, maybe you mean `oneline` ?